modules/developers-guide/pages/webpack-config.adoc

```diff
@@ -5,46 +5,101 @@ The tutorials introduced a few basic approaches to building a front-end user int
 For example, the tutorials demonstrated:
 
 * Using link:tutorials/hello-location{outfilesuffix}#candid-ui[Candid] as a bare-bones interface to expose and test the functions in a canister.
-* Using link:tutorials/explore-templates{outfilesuffix}#default-frontend[raw JavaScript] to display a simple pop-up alert window.
+* Using link:tutorials/explore-templates{outfilesuffix}#default-frontend[raw HTML and JavaScript] to display a simple HTML entry page.
 * Using link:tutorials/custom-frontend{outfilesuffix}[React and compiled JavaScript] to embed HTML attributes and elements directly in a page.
 * Using link:tutorials/my-contacts{outfilesuffix}[React and TypeScript] to import CSS properties from an external file.
 
 This section takes a closer look at the default front-end template, front-end configuration options, and using other frameworks to build the user interface for your projects.
 
-== How the default template is used
+== How the default templates are used
 
-As you might have noticed in the tutorials, projects include a template `+webpack.config.js+` file by default.
-Because webpack is a popular and highly-configurable module bundler for JavaScript-based applications, the default `+webpack.config.js+` file makes it easy to add the specific modules—such as `react` and `markdown`—that you want to use.
+As you might have noticed in the tutorials, projects include template `index.js` and `+webpack.config.js+` files.
 
-By default, the template `+webpack.config.js+` file also creates canister aliases based on the entries in the project's `+dfx.json+` file.
-If you review the code in the template `+webpack.config.js+` file, you see that it performs this alias creation task using the following code block:
+By default, the `+index.js+` file explicitly instantiates an instance of the HTTP agent. This agent provides an interface for interactions between the user-facing front-end and the {IC}. 
+After creating the agent, the `+index.js+` file constructs an actor for the canisters defined in your project's `+dfx.json+` file.
+
+If you review the code in the `+index.js+` file, you see that it performs these steps using the following code block:
+
+[source,js]
+----
+import { Actor, HttpAgent } from '@dfinity/agent';
+import { idlFactory as hello_idl, canisterId as hello_id } from 'dfx-generated/hello';
+
+const agent = new HttpAgent();
+const hello = Actor.createActor(hello_idl, { agent, canisterId: hello_id });
+----
+
+After the agent and the actor are created for the project, the index.js provides a placeholder for the JavaScript you use to interact with the document object model (DOM) and HTML for your application.
```

Comment:
Nit: "...the project, the index.js..." add "file" after index.js or get rid of "the".

Reply:
fixed.